funsite/templates/lms/register.html

```diff
@@ -159,6 +161,12 @@ <h2 class="color-regal-blue">
                             link_start='<a href="{url}"  tabindex="-1">'.format(url=honor_code_path),
                             link_end='</a>')}</label>
                     </div>
+
+                    <div class="form-group" id="field-newsletter">
+                        <input id="newsletter" type="checkbox" value="true" />
```

Comment:
I do not understand the purpose of `value=true`
```suggestion
                        <input id="newsletter" type="checkbox" />
```